releasenotes/notes/releasenotes-946616ca3ce71d03.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Scrub pod probes http headers and command.
```

Comment:
```suggestion
    The orchestrator check can now scrub sensitive data from probes in pods specifications.
```

Reply:
should probably also be in the dca release notes

Reply:
Done